lib/elixir/lib/system.ex

```diff
@@ -880,6 +880,52 @@ defmodule System do
     :init.stop(String.to_charlist(status))
   end
 
+  @doc ~S"""
+  Executes the given `command` in the current OS shell.
+
+  **Important**: Use this function with care. In particular, **never
+  pass user input to this function**, as the user would be able to
+  execute any code directly on the machine. Generally speaking,
```

Comment:
I think it would be good to mention that this is a "command injection" vulnerability, so the reader has a keyword to search for and better understand the implications.